ntbs-service-unit-tests/DataMigration/ImportValidatorTest.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using ntbs_service.DataAccess;
+using ntbs_service.DataMigration;
+using ntbs_service.Models.Entities;
+using ntbs_service.Models.Enums;
+using ntbs_service.Models.Validations;
+using Xunit;
+
+namespace ntbs_service_unit_tests.DataMigration
+{
+    // This test suite is not designed to test that the validation is correct, only that the validation is happening on
+    // all parts of a notification that we expect it to happen on - which is why it isn't testing many validation cases.
```

Comment:
This comment is useful, but I think it would be better to rephrase it to show that we're not happy with the status quo (i.e. it would be better if these tests did test the validation more fully). Basically, I'd like the comment to say something along the line of 'this is what we've done so far, but there's more work to be done'.